README.md

```diff
@@ -22,7 +22,7 @@ Next, start an MLLP.Sender process and store its PID.
 
 Alternatively, you could start a Sender using a DNS name rather than an IP address.
 ```
-{:ok, s1} = MLLP.Sender.start_link("localhost", 4090)
+{:ok, s1} = MLLP.Sender.start_link("localhost", port: 4090, MLLP.EchoDispatcher)
```

Comment:
We don't need dispatched on `Sender.start_link`

Reply:
Also the port stays as parameter on Sender

Reply:
Good catch.